src/main/io/osd.h

```diff
@@ -160,6 +160,7 @@ typedef enum {
     OSD_WARNING_BATTERY_CRITICAL,
     OSD_WARNING_VISUAL_BEEPER,
     OSD_WARNING_CRASH_FLIP,
+    OSD_WARNING_CONSTANT_IDLE,
```

Comment:
The new warning enum should be added at the end.  Otherwise it'll be more work in the configurator to rearrange the warnings order.  Support for this in the configurator is needed anyway, but it's less work to simply add a new warning rather than rearrange everything.

Also the bigger issue is that changing the order corrupts the bitpacked values so it will break any previous configurations.

Reply:
fixed